src/Proto.Actor/Scheduler.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Proto
+{
+    public static class Scheduler
+    {
+        public static void ScheduleTellOnce(this IContext context, TimeSpan delay, PID target, object message)
+        {
+            Task.Run(async () =>
+            {
+                await Task.Delay(delay);
+
+                context.Tell(target, message);
```

Comment:
Another question that we need to decide on, should scheduled messages go through the current actors middleware?
(I would say yes, but there might be more to this that needs to be explored)